CampaignConfig/mdc2020_primary.cfg

```diff
@@ -14,21 +14,18 @@ digidb_version = v1_1
 recodb_version = v1_1
 db_purpose = best
 data_tier = dts
-outdir = /pnfs/mu2e/tape/phy-sim/dts/mu2e/
 stopped = Muminus
 evts = 2000
 primary_name = NoPrimary
 digitype = OnSpill
-beam = 2BB
+beam = 1BB
```

Comment:
We eventually need to have 1BB and 2BB NoPrimary.  Maybe they should be separate campaigns?